packages/bpk-component-phone-input/readme.md

```diff
@@ -0,0 +1,67 @@
+# bpk-component-phone-input
+
+> Backpack phone input component.
+
+## Installation
+
+```sh
+npm install bpk-component-phone-input --save-dev
+```
+
+## Usage
+
+```js
+import React, { Component } from 'react';
+import BpkPhoneInput from 'bpk-component-phone-input';
+
+export default class extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { dialingCode: '44', phoneNumber: '' }
+  }
+
+  onChange = (dialingCode, phoneNumber) => {
+    this.setState({ dialingCode, phoneNumber });
+  }
+
+  render() {
+    return (
+      <BpkPhoneInput
+        id="phone-input-id"
+        name="Telephone input"
+        placeholder="Telephone number"
+        onChange={this.onChange}
+        value={this.state.phoneNumber}
+        dialingCode={this.state.dialingCode}
+        dialingCodes={[
+          { code: '44', description: '+44' },
+          { code: '55', description: '+55' },
+        ]}
+        dialingCodeProps={{
+          id: 'dialing-code',
+          name: 'Dialing code',
+          'aria-label': 'Dialing code',
+        }}
+      />
+    )
+  }
+}
+```
+
+## Props
+
+| Property              | PropType                                              | Required   | Default Value    |
+| --------------------- | ----------------------------------------------------- | ---------- | ---------------- |
+| dialingCode           | string                                                | true       | -                |
+| dialingCodeProps      | shape({ id: string, name: string })                   | true       | -                |
+| dialingCodes          | arrayOf(shape({ code: string, description: string })) | true       | -                |
+| id                    | string                                                | true       | -                |
+| name                  | string                                                | true       | -                |
+| id                    | string                                                | true       | -                |
+| onChange              | func                                                  | true       | -                |
```

Comment:
Should we have two onChange functions instead of one? Personally think it'd be nicer with one for each case e.g

+ `onNumberChange`
+ `onDialingCodeChange`

Reply:
We could, although I'd prefer `onChange` and `onDialingCodeChange` in this case. Not sure what would be the advantage of that though.